spot-contracts/contracts/_interfaces/IFeePolicy.sol

```diff
@@ -41,34 +16,23 @@ interface IFeePolicy {
     /// @return The deviation ratio given the system subscription parameters.
     function computeDeviationRatio(SubscriptionParams memory s) external view returns (uint256);
 
+    /// @param seniorTR The senior tranche ratio of the deposit bond.
+    /// @return The "targetSR" adjusted senior tranche ratio,
+    ///         as a fixed-point number with {DECIMALS} decimal places.
+    function computeDRNormSeniorTR(uint256 seniorTR) external view returns (uint256);
+
+    /// @notice Computes magnitude and direction of value flow between perp and the rollover vault
+    ///         expressed in underlying tokens.
     /// @param s The subscription parameters of both the perp and vault systems.
-    /// @return r Rebalance data, magnitude and direction of value flow between perp and the rollover vault
-    ///           expressed in the underlying token amount and the protocol's cut.
-    function computeRebalanceData(SubscriptionParams memory s) external view returns (RebalanceData memory r);
+    /// @return underlyingAmtIntoPerp The value in underlying tokens, that need to flow from the vault into perp.
+    function computeRebalanceAmount(SubscriptionParams memory s) external view returns (int256 underlyingAmtIntoPerp);
+
+    /// @return The share of the system tvl paid to the protocol owner as fees.
```

Comment:
per rebalance period?

A reader could easily interpret this as an annual amount.